spring-jdbc/src/main/java/io/crate/spring/jdbc/samples/ImageNotExistsAdvice.java

```diff
@@ -0,0 +1,24 @@
+package io.crate.spring.jdbc.samples;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ControllerAdvice
+public class ImageNotExistsAdvice {
+
+    @ResponseBody
+    @ExceptionHandler(ImageNotExistsException.class)
+    @ResponseStatus(HttpStatus.NOT_FOUND)
+    Map<String, Object> imageNotFoundHandler(ImageNotExistsException ex) {
```

Comment:
Why don't we introduce a datamodel instead of working with untyped Maps ?

Reply:
Does this really work with the existing frontend? I didn't want to invest here